### ISSUE TITLE
Configuraton/Recipe Help(?) (Matter Overdrive)

### ISSUE THREAD

**Issue by MonfGeiger** [DarkPacks/SkyFactory-4#2281]
Most of what i'm about to talk about relates to this post here: #2083

The TL;DR is i decided to, for the heck of it, do SF4 in a single player run after importing the modpack to MultiMC, but i DID remember the jankiness from the matter overdrive legacy edition mod. Before i discovered the aforementioned issue/topic i referenced above, I saw there was a community edition version that both fixed stuff the legacy edition left bugged among other things. I FIGURED that all i needed to do was to just drop it into the mods folder, and disable the legacy edition version of matter overdrive and that would be it. So i do, and i load up a new world.....yet, when i checked the recipe list for dirt resin, there was only ONE recipe (i think it was for creating a slimy piston?). The community edition basically broke the modpack as it is today. I can fix this by simply disabling the Community Edition of MO and re-enabling the Legacy Edition of MO, so as far as **bug reports** go.....this one is pretty much self-solved already

I know its been nearly two years since the referenced issue/mod request was done, so unless i would be better off asking/requesting this next part from a different forum somewhere (or the ftb reddit, or a different reddit?), could i ask for assistance on adjusting the modpack (and whatever configuration file(s) it uses for recipes) so it can run off the community edition of MO, please?

## World Information
* World Preset: Default (with cheats enabled)
* Prestige Enabled: None
* Modpack Version world created in: 4.2.4
* Additional Content Installed: Matter Overdrive (Community Edition. Currently Disabled in MultiMC)

## Client Information
* Modpack Version: 4.2.4
* Java Version: jre1.8.0_351
* Launcher Used: MultiMC 5, version 0.7.0-3826 (dev release)
* Memory Allocated: 
* Server/LAN/Single Player: Min: 2 GB, Max: 9 GB
* Optifine Installed: No
* Shaders Enabled: No (God no, my desktop wouldn't be able to handle it, ill take things with their default look)
